src/server/error-controller.js

```diff
@@ -36,7 +36,7 @@ class ErrorController {
 
       if (isApiCall) {
         this.sendErrorJson(res, consolidatedErr);
-      } else if (!this.tryRedirectToLogin(req, res, consolidatedErr)) {
```

Comment:
This is already called a few lines above

Reply:
Indeed! 😳